tests/foundry/koverage/AssertTest.test_bool.expected

```diff
@@ -0,0 +1,6 @@
+{
+  false
+#Equals
+  VV0_x_114b9705 ==Int 0 ==Bool true ==Bool true
+}
+
```

Comment:
These tests data need to be moved to the new location, at `kevm-pyk/src/tests/integration/test-data` instead.